### PR TITLE
Add tests/data/README.md

### PR DESCRIPTION
**Pull request by y-guyon**

### CODE REVIEW

File: tests/data/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+License: [Creative Commons Attribution license (reuse allowed)](https://creativecommons.org/licenses/by/3.0/legalcode)
+
+Source: ["Cosmos Laundromat" movie](https://www.youtube.com/watch?v=Y-rmzh0PI3c) (frame at around 1 minute past the
+beginning)
+
+## Metadata
+
+The structure can be displayed using `exiv2 -pS <file>`.
+
+### File `paris_exif_xmp_icc.jpg`
```
Comment (jzern) [inline tests/data/README.md:22]: these could be local links, but that may only make sense if the generated file is output to this directory

**Review by jzern — APPROVED** [review]

**Review by wantehchang — APPROVED** [review]
LGTM.

File: tests/data/README.md (hunk 1 above)
Comment (y-guyon) [reply tests/data/README.md]: I used local links and also inline images.

File: tests/data/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Test images
+
+## Still images
+
+### Files `kodim*`
+
+License: released by the Eastman Kodak Company for unrestricted usage
+
+Source: http://r0k.us/graphics/kodak/
```
[the remark this answers is not in the document]
Comment (y-guyon) [reply tests/data/README.md]: I tried `pandoc -f markdown+autolink_bare_uris -t markdown tests/data/README.md -o tests/data/README.md` but it changed a lot of things. I just added `<>` to the link.

[the remark this answers is not in the document]
Comment (jzern) [reply tests/data/README.md]: Thanks for checking. I guess I only looked at the one link and didn't check for other issues.

File: tests/data/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+Source: <http://r0k.us/graphics/kodak/>
+
+### Files `cosmos*`
+
+![](io/cosmos1650_yuv444_10bpc_p3pq.avif)
+
+License: [Creative Commons Attribution license (reuse allowed)](https://creativecommons.org/licenses/by/3.0/legalcode)
+
+Source: ["Cosmos Laundromat" movie](https://www.youtube.com/watch?v=Y-rmzh0PI3c) (frame at around 1 minute past the
```
Comment (jzern) [inline tests/data/README.md:21]: It isn't critical, but I realized you could give a time offset here to bring the clip somewhat close to where you wanted it:
https://youtu.be/Y-rmzh0PI3c?t=60

**Pull request merged**